src/main/java/seedu/guest/model/guest/Guest.java

```diff
@@ -21,12 +21,13 @@ public class Guest {
     private final NumberOfGuests numberOfGuests;
     private final IsRoomClean isRoomClean;
     private final Bill bill;
+    private final Request request;
 
     /**
      * Every field must be present and not null.
      */
     public Guest(Name name, Phone phone, Email email, Room room, DateRange dateRange,
-                 NumberOfGuests numberOfGuests, IsRoomClean isRoomClean, Bill bill) {
+                 NumberOfGuests numberOfGuests, IsRoomClean isRoomClean, Bill bill, Request request) {
         requireAllNonNull(name, phone, email, dateRange, numberOfGuests, isRoomClean, bill);
```

Comment:
Should request be here as well?

Reply:
Should, because it need to mark whether the guest have request
if no request, I set as "-"